django/core/views.py

```diff
@@ -447,6 +450,12 @@ class EventViewSet(CommonViewSetMixin, OnlyObjectPermissionModelViewSet):
         "start_date",
     )
 
+    def get_queryset(self):
+        # exclude spam from list view
+        if self.action == "list":
+            return self.queryset.public()
```

Comment:
am I understanding this correctly to allow possible spam events to be accessible in the detail view? (same for Jobs?)

Reply:
Yes, mostly to give an explanation of what happened should a real post get mistakenly flagged. Though it should probably only be accessible by admins and the submitter